index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms">
+    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <ul class="form_field">
+        <li class="form_name">
+            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name">
+        </li>
+        <li class="form_email"> 
+            <input type="email" id="email" name="email" class="input" placeholder="Email Address">
+        </li>
+        <li class="form_text">
+           <textarea name="text" id="message" cols="32" rows="6" placeholder="Entre text here"></textarea>
```

Comment:
- [ ] Kindly make the message field accepts up to 500 chars and adds `required` attribute.

Reply:
okay, good